src/corecel/Assert.cc

```diff
@@ -220,8 +220,7 @@ RuntimeError RuntimeError::from_geant_exception(const char* origin,
  * Construct an error message from a Geant4 exception.
  *
  * \param origin Usually the function that throws
- * \param code A computery error code
```

Comment:
computery :)

Reply:
In Geant4 they're arbitrary strings that look like fortran error codes: not human readable, not computer readable... just kinda "computery" 😉